pkgs/by-name/gp/gpac/package.nix

```
@@ -5,35 +5,97 @@
   cctools,
   pkg-config,
   zlib,
+  ffmpeg,
```

This only uses headless, right?

```suggestion
  ffmpeg-headless,
```